source/_docs/free-https.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Early Access: Free and Automated HTTPS
+earlyaccess: true
+description: Upgrade to Free and Automated HTTPS, powered by Let's Encrypt
+---
+Upgrade your sites's HTTPS and never manage certificates again. Pantheon automatically adds all of your site's domains to a shared certificate and  serves it through our globally distributed content delivery network (CDN). Upgrade to take advantage of performance, security benefits, plus the time and cost of manually purchasing and maintaining certs.
+
+## Eligibility
+Invite only. As of April 2017, sites on a Professional plan with HTTPS already enabled and all custom domains routing to Pantheon may [request an invite](http://learn.pantheon.io/201701-HTTPS-Reg.html).
+
+## Upgrade Your Site
+
+1. Click the **Start HTTPS Upgrade** button from the Site Dashboard.
+2. Wait 25 to 60 minutes for HTTPS provisioning to complete.
+3. Once HTTPS is ready, you will see action required on the **Domains & HTTPS** tab for each environment, with instructions to configure DNS for each custom domain.  
+4. Click **Show DNS Recommendations** next to each custom domain to identify DNS values needed to point the domain to your site. Domains that are not yet configured will indicate action is required.
+5. Configure DNS using the provided destinations at the domain's DNS provider.
+
+    <div class="alert alert-info">
+    <h3 class="info">Pro Tip</h3>
+Look up your DNS provider with this free web tool: <a href="https://mxtoolbox.com/DNSLookup.aspx">https://mxtoolbox.com/DNSLookup.aspx</a>
+</div>
+
+6. Wait for your DNS changes to fully propagate. DNS records are cached across the internet and can take up to 72 hours to propagate, depending on the time to live (TTL) configured for the domain's DNS records.
```

Comment:
Cool. I like the wording here a little better: "can take up to 72 hours to go into effect, but typically they happen much sooner."

I just don't want anything to think they *need* to wait, especially if the tool below shows global propagation.